constants.go

```diff
@@ -83,12 +83,6 @@ const (
 	// at least a fixed number of traces per second.
 	SamplerTypeLowerBound = "lowerbound"
 
-	// DefaultUDPSpanServerHost is the default host to send the spans to, via UDP
-	DefaultUDPSpanServerHost = "localhost"
-
-	// DefaultUDPSpanServerPort is the default port to send the spans to, via UDP
-	DefaultUDPSpanServerPort = 6831
-
```

Comment:
Removing these public constants is a breaking change, please keep them here. Our approach has been that submodules like `util` should not contain their own defaults (as much as possible).

Reply:
It kind of makes sense for the constant to be accessible from that util file since the constructor can be called directly. But if the top level package is included, there will be an import cycle

Reply:
Maybe i could duplicate the constants into a package?

Reply:
Why do you need these constants in util? If util does not do its own defaulting (and it shouldn't), then the values of these constants are irrelevant inside util.

Reply:
Ok i've removed these constants from util